mediorum/server/server_basic_auth.go

```diff
@@ -20,6 +22,12 @@ func parsePrivateKey(pk string) (*ecdsa.PrivateKey, error) {
 	return crypto.ToECDSA(privateBytes)
 }
 
+// From https://github.com/AudiusProject/sig/blob/main/go/index.go
+func recover(input string, signature []byte) (common.Address, error) {
+	hash := crypto.Keccak256Hash([]byte(input))
+	return sigverify.EcRecoverEx(hash.Bytes(), signature)
+}
+
```

Comment:
probably want to put this into its own util file if the Sig repo linked here won't have packages to import